decode/src/datapath.rs

```diff
@@ -194,7 +194,20 @@ impl schemars::JsonSchema for EthernetComplianceCode {
     }
 
     fn json_schema(gen: &mut schemars::gen::SchemaGenerator) -> schemars::schema::Schema {
-        String::json_schema(gen)
+        let mut s = String::json_schema(gen);
```

Comment:
At this point I'd just construct it my hand rather than modifying the String schema, but ... meh. either way.